ports/libzippp/portfile.cmake

```
@@ -1,9 +1,9 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO ctabin/libzippp
-    REF 791bdc43eb18b87e3bdfa087493e3e32217e672c #v4.0-1.7.3 with CXX std version c++11
-    SHA512 c6a90ecec21bb2d9e3af681c35d7eec0bee7b356fc1438004dc84be32ee7b94d047c35817d46b222237d54699ea54afa4fd3ae5deeba40dfce4fd2035a38b0e5
-    HEAD_REF libzippp-v4.0-1.7.3
+    REF 8299422194ce3c5be0677550ce3d6d4e15d40dd8 #v4.1-1.8.0 with CXX std version c++11
+    SHA512 091c744377707899456b027a35059e048e1552e013330c68920f88f94a42904cf1a6afc7f853cf34cc9bbb3956c3c16907f089520e808520a671c59646d5e351
+    HEAD_REF libzippp-v4.1-1.8.0
```

```suggestion
    HEAD_REF master
```

This should be the ref used to build the latest head, not the current release.

---

Looks strange to me, because the commit really points to the tag libzippp-v4.1-1.8.0. Anyway, I changed the ref to the master.